game_options.cpp

```diff
@@ -43,7 +63,7 @@ void test_game_options()
   // Music can be switched off
   {
     const bool do_play_music = false;
-    const game_options g(do_play_music);
+    const game_options g(0, do_play_music);

```

Comment:
This edit scared me a bit, as the first impression would be that only the first player (with index 0) has no sound :-). I would have appreciated a named constant instead of the zero:

```c++
const int rng_seed = 0;
const game_options g(rng_seed, ...
```

Still, with minimal effort I could find out that zero is the RNG seed, so I will accept. I only want to share that named constants helps understand the code even better :-)

Reply:
Definitely agree with this, I was unsure how to proceed as I'm used to R where you can just provide arguments as name-value pairs on a function call. I like the idea of declaring a variable just above for better readability!